Terminal.Gui/View/View.Hierarchy.cs

```diff
@@ -7,43 +7,81 @@ namespace Terminal.Gui;
 public partial class View // SuperView/SubView hierarchy management (SuperView, SubViews, Add, Remove, etc.)
 {
     [SuppressMessage ("Style", "IDE1006:Naming Styles", Justification = "<Pending>")]
-    private static readonly IList<View> _empty = new List<View> (0).AsReadOnly ();
+    private static readonly IReadOnlyCollection<View> _empty = [];
 
     private List<View>? _subviews; // This is null, and allocated on demand.
 
-    // Internally, we use InternalSubviews rather than subviews, as we do not expect us
-    // to make the same mistakes our users make when they poke at the Subviews.
-    internal IList<View> InternalSubviews => _subviews ?? _empty;
+    // Internally, we use InternalSubViews rather than subviews, as we do not expect us
+    // to make the same mistakes our users make when they poke at the SubViews.
+    internal IList<View> InternalSubViews => _subviews ?? [];
```

Comment:
I think that `InternalSubViews` also can be `IReadOnlyCollection` instead of `IList`.

Reply:
> I think that `InternalSubViews` also can be `IReadOnlyCollection` instead of `IList`.

No. It's bad that internal code uses _subviews directly. See latest push which makes all internal usages use InternalSubViews and it's def not read only.